dora-parser/src/ast.rs

```diff
@@ -511,6 +529,18 @@ pub struct Module {
     pub initializers: Vec<Box<Stmt>>,
 }
 
+#[derive(Clone, Debug)]
+pub struct Annotation {
+    pub id: NodeId,
+    pub name: Name,
+    pub pos: Position,
+    pub annotation_usages: AnnotationUsages,
+    pub internal: bool,
```

Comment:
I believe this shouldn't be `bool` but e.g. `Option<InternalAnnotation>`. `InternalAnnotation` would be enum that has all internal known annotations, so `enum InternalAnnotation { Open, Static, ....}`.